memo/Cargo.toml

```diff
@@ -3,15 +3,15 @@
 
 [package]
 name = "spl-memo"
-version = "0.1.0"
+version = "1.0.0"
 description = "Solana Program Library Memo"
 authors = ["Solana Maintainers <maintainers@solana.com>"]
```

Comment:
Oh, we should mirror this commit in SPL: https://github.com/solana-labs/solana/commit/48febf153bfb176e048b727be35966075da10dd6#diff-474f9d160624516e523edbad6af0815f

Reply:
@mvines Good point, I'll make those changes.  We should also update the other examples as well correct?

Reply:
https://github.com/solana-labs/solana-program-library/pull/42